activities/QRCode.activity/js/activity.js

```diff
@@ -38,7 +38,6 @@ define(["sugar-web/activity/activity","sugar-web/datastore", "sugar-web/env", "w
 		var generateCode = function(text) {
 			qrCode.clear();
 			qrCode.makeCode(text);
-			var text = userText.value.toLowerCase();
```

Comment:
I think removing the toLowerCase is not the solution else the following line will not be able to detect string like "HTTP://" or "HttPs://". The solution is just to put the raw value in history.